.github/workflows/tests.yml

```diff
@@ -127,3 +127,13 @@ jobs:
         with:
           name: mutants.out
           path: mutants.out
+
+  cffconvert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - uses: docker://citationcff/cffconvert:latest
+        with:
+          args: --validate
```

Comment:
This is the basic validation setup.  It shall ensure that any changes to the CITATION.cff do not affect its validity.

After the mandatory repository checkout, the official validator is called on the file.  The CFF team provides a Python 3 CLI for conversion and validation purposes which is also available as this Docker image.  As the version tag is set to `latest`, you will always have the latest available version of the validator checking the configured CITATION.cff.  Thus, this logic does not need any maintenance -- except for upcoming major updates to actions/checkout.

Reply:
The Docker image usually succeeds within about 10 seconds which is why I added it to the main testing CI.